### ISSUE TITLE
Colour codes included in slack message

### ISSUE THREAD

**Issue by JaniJegoroff** [KrauseFx/fastlane#48]
Looks like exception message comes through with colour codes. See **[33m** and **[0m** in the screenshot:

![fastlane-slack-color-code](https://cloud.githubusercontent.com/assets/2363231/6214595/97a95eb4-b5f1-11e4-855c-eb926cfa8b4b.png)

Can we somehow prevent it so that STDOUT prints are still coloured?

**Steps to reproduce**
Given I have following Fastfile defined:
```Ruby
before_all do
  ENV['SLACK_URL'] = <my-slack-hook>
end

lane :preview do
  deliver :skip_deploy
end

error do |lane, exception|
  slack({
    message: exception.message,
    success: false,
    channel: <my-slack-channel>
  })
end
```
When I execute `bundle exec fastlane preview`
And I answer **n** to `Does the PDF on path '/tmp/1424100823.pdf' look okay for you? (blue = updated) (y/n)`
Then I should receive slack message without colour codes:
![fastlane-slack-no-color-code](https://cloud.githubusercontent.com/assets/2363231/6214766/efdba46a-b5f2-11e4-8cc9-eee0d59c3ad1.png)

**Workaround**
Right now I have workaround in Fastfile which does the job:
```Ruby
error do |lane, exception|
  slack({
      message: exception.message.gsub(/\033\[\d+m/, ''),
      success: false,
      channel: <my-slack-channel>
    })
end
```

**Comment by KrauseFx** [comment]
That's my fault, I have some code like https://github.com/KrauseFx/fastlane/blob/master/lib/fastlane/fast_file.rb#L8, which adds the `.red` to the error messages.

This was used across all other tools as well.

I like the idea of showing the error message inside the slack message.

How would you suggest can we fix this problem?

**Comment by JaniJegoroff** [comment]
Yeah, it's a side effect of using gems like `colored`. I had initial look yesterday but could not find a proper solution (yet) where messages are coloured in the console but "plain" inside exceptions.

**Comment by KrauseFx** [comment]
How about for now, just pass a third parameter, which is the error string without the color code, using the code you posted.

**Comment by JaniJegoroff** [comment]
> How about for now, just pass a third parameter, which is the error string without the color code, using the code you posted?

Suits for me.

**Comment by JaniJegoroff** [comment]
Fixed in release 0.4.1
